A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2321,12 +2322,10 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     getVLS() {
-        // for this to be implemented a FLAPS 3 landing logic is needed.
-        /*const flapsHandleIndex = Simplane.getFlapsHandleIndex();
```

Comment:
This is enabling some existing code that was commented out. I haven't verified the calculations against a manual yet but they seem reasonable in flight.

Reply:
The calculation used is accurate.

Reply:
Thanks. This code should be cleaned up in future then?

Reply:
If no other usage, feel free to remove otherwise replace with the correct calculation/output.